src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -91,7 +91,18 @@
     </None>
     <None Include="Xamarin.Android.Bindings.targets">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-      <SubType>Designer</SubType>
+    </None>
+    <None Include="Xamarin.Android.Bindings.ClassParse.targets">
```

Comment:
Should these added files instead go into: https://github.com/xamarin/xamarin-android/tree/master/src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android

Doing so helps reduce the "file pollution" going in in `src/Xamarin.Android.Build.Tasks`, which makes *me* happier when doing `code -n src/Xamarin.Android.Build.Tasks/*.targets`…